Makefile

```diff
@@ -103,6 +103,20 @@ deploy-example: deploy
 	@oc create -n openshift-logging -f hack/cr.yaml
 .PHONY: deploy-example
 
+deploy-example-secret:
+	mkdir /tmp/example-secrets && \
+	hack/cert_generation.sh /tmp/example-secrets openshift-logging elasticsearch && \
+	  oc create secret generic elasticsearch -n openshift-logging \
```

Comment:
we have similar logic in `hack/deploy-setup.sh` would it make sense to build off that script for creating the secret at all?

Reply:
Ideally i believe yes but the deploy-setup doesnt have any value because you can not use 'make run' to run locally.  The local operator has no way to resolve ES running on the cluster.  We would need to figure that out.

Reply:
> The local operator has no way to resolve ES running on the cluster

isn't that what your kubeconfig is for?

Reply:
The operator essentially does a `curl` of the ES endpoint which would run 'locally'.  your local machine cant resolve the internal cluster ips

Reply:
ah, ack... you could always manually update your `resolv.conf` to point to the cluster for DNS, or even update your `hosts` file.